chain-abci/tests/validator_changes.rs

```diff
@@ -145,20 +146,20 @@ fn check_unbonding_with_removing_validator() {
 #[test]
 fn check_rejoin() {
     // Init Chain
-    let (env, storage) = ChainEnv::new_with_customizer(
+    let (mut env, storage) = ChainEnv::new_with_customizer(
         (Coin::max() / 2).unwrap(),
         (Coin::max() / 2).unwrap(),
         2,
         |parameters| {
             // tweaking times + parameters, something more than 0.0... gets minted
-            parameters.unbonding_period = 3600 * 24 * 10000;
             parameters.rewards_config.reward_period_seconds = 365 * 24 * 3600;
             parameters.required_council_node_stake = (Coin::max() / 4).unwrap();
             parameters.rewards_config.monetary_expansion_r0 = Milli::try_new(1, 0).unwrap();
             parameters.rewards_config.monetary_expansion_tau = 1000_0000_0000_0000_0000;
             parameters.rewards_config.monetary_expansion_decay = 0;
         },
     );
+    env.max_evidence_age = 3600 * 24 * 10000;
```

Comment:
`unbonding_period` was in seconds and `max_evidence_age` is in nanoseconds. I'm not sure if this is correct? Check this at other places also.

Reply:
Yes, but we only take the seconds part of evidence age duration in app. This is handled in init chain event.

Reply:
https://github.com/crypto-com/chain/blob/master/chain-abci/src/app/app_init.rs#L400